File: corpus/py/F;0405.py
private static Object teclado
String n = null
String resultado
int i = 0
while i<15:
    PreencherNome(vetNome)
    PreencherAcento(vetAcento)
    resultado = pesquisarAcento(vetNome, vetAcento, n)
    print("O nome do passageiro é:" + vetNome[i])
    print("O acento desse passageiro é:" + resultado)
    i ++
def PreencherNome(String vnome[]):
    int i = 0
    print("Digite o nome do passageiro:")
def PreencherAcento(String vacento[]):
    int i =0
    print("Digite o acento:")
private static String pesquisarAcento(String[] vetorNome, String[] vetorAcento, String nome) {
int i =0
if i<=0:
return nome

File: corpus/2018q1/_p2q2_ok/_check/F;0405.java

import java.util.Scanner;

public class QUESTAO2 {
    private static Object teclado;

    public static void main(String[] args) {

        String vetNome [] = new String[15];
        String vetAcento [] = new String [15];
        String n = null;
        String resultado; 
        int i = 0;

        while(i<15){

        PreencherNome(vetNome);
        PreencherAcento(vetAcento);

        resultado = pesquisarAcento(vetNome, vetAcento, n);

        System.out.println("O nome do passageiro é:" + vetNome[i]);
        System.out.println("O acento desse passageiro é:" + resultado);

        i++;
    }

    public static void PreencherNome (String vnome[]){

        Scanner teclado = new Scanner(System.in);

       int i = 0;

            System.out.println("Digite o nome do passageiro:");
            vnome[i] = teclado.nextLine();
        }

    public static void PreencherAcento (String vacento[]){

        Scanner teclado2 = new Scanner(System.in);

         int i =0;

            System.out.println("Digite o acento:");
             vacento[i] = teclado2.nextLine();

    }

    private static String pesquisarAcento(String[] vetorNome, String[] vetorAcento, String nome) {

        int i =0;

       if(i<=0){
           nome= vetorAcento[i];
        }
    return nome;
    }

}

}
